CHANGELOG

```diff
@@ -13,7 +13,7 @@ Agenda:
 - Changed API of related objects. All assignments, motions and custom slides
   are now agenda items and can be hidden.
 - Removed mptt.
-- Added attachments to custom slides.
+- Added attachments to agenda items.
```

Comment:
Mmh that's not really from user view - because user know only agenda items, motions and assignments. No custom slides.So maybe we should change the item to lines before...